### ISSUE TITLE
Disabling arrow key events

### ISSUE THREAD

**Issue by antonio-gomez** [Foliotek/Croppie#191]
Is there a way I can disable the arrow key events? Maybe as a parameter passed that disables the Croppie viewport position being affected?

Thanks!

**Comment by thedustinsmith** [comment]
No, sorry, that's not built into the API.  However, I can see that being a useful addition.  If you want to submit a PR, I'd be happy to merge it in.

**Comment by michelcve** [comment]
I believe this issue can be closed, JavierAroche fixed this a long time ago.

**Comment by antonio-gomez** [comment]
Thanks @JavierAroche
